scripts/deploy-k8s-components.sh

```diff
@@ -2,10 +2,10 @@
 set -ex
 
 ROOT_DIR=$(dirname $0)/..
-NAMESPACE=${NAMESPACE:-examples}
+NAMESPACE=${NAMESPACE:-object-detection}
```

Comment:
Revert this change.

Reply:
made a separate folder for original scripts